### DISCUSSION TITLE
Context menu z-index problems

### DISCUSSION THREAD

**Discussion by elipp** [josdejong/svelte-jsoneditor#127] [Q&A] [answered]
Currently integrating svelte-jsoneditor to a Django admin view, and the context menu likes to clip itself as shown in the screenshot:

![image](https://user-images.githubusercontent.com/1861512/185083718-e2781884-00a1-40ce-8d98-b4a2e8856d26.png)

How does one avoid this?

**Comment by josdejong** [comment, marked as answer]
This happens when the editor is created inside a DIV that has styling `overflow: hidden`.

It's is often used to prevent the contents of a DIV from overflowing, but instead of you can use flexbox and specify `min-width: 0` and/or `min-height: 0`  to prevent any overflowing content.

**Reply by elipp** [reply]
Thanks for the fair reply to what was, frankly, a rather basic question ^^
